onnxruntime/test/ir/graph_test.cc

```diff
@@ -1807,7 +1808,13 @@ TEST_F(GraphTest, InjectExternalInitializedTensors) {
 
   const TensorProto* external_data = nullptr;
   ASSERT_TRUE(graph.GetInitializedTensor(initializer_name, external_data));
-  ASSERT_TRUE(utils::HasExternalData(*external_data));
+  if (external_data) {
+    ASSERT_TRUE(utils::HasExternalData(*external_data));
+  } else {
+    // Fail the test because external_data shouldn't be nullptr.
+    // This if-else is added for suppressing warning C6011: dereferencing NULL pointer.
+    ASSERT_TRUE(false);
```

Comment:
Perhaps, ASSERT_NOT_NULL for external_data would be less wordy and more descriptive.

Reply:
```cpp
ASSERT_NOT_NULL(ptr) => This pointer shouldn't be null.
ASSERT_TRUE(false) => This case should never happen.
```
The later one is closer to what I want to express.